OrbitCore/StringManager.cpp

```diff
@@ -28,3 +28,9 @@ void StringManager::Clear() {
   absl::MutexLock lock{&mutex_};
   key_to_string_.clear();
 }
+
+std::vector<std::pair<uint64_t, std::string>> StringManager::ToVector() {
```

Comment:
I suppose this is necessary because absl::flat_hash_map is not directly supported by cereal? Maybe add a comment in the header that this method exists to support serialization (and shouldn't really be used outside of that)?

Reply:
Actually things seem to work as expected with cereal so I switched to the usual `ORBIT_SERIALIZABLE`/`ORBIT_SERIALIZE` macros.